src/components/TopSlider/TopSlider.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect } from 'react';
+import { AutoRotatingCarousel } from 'material-auto-rotating-carousel';
+import { Slide } from 'material-auto-rotating-carousel';
```

Comment:
Just wanted to pay your attention, that we can do these imports in one line.

Reply:
Wat?

Reply:
import { AutoRotatingCarousel, Slide } from 'material-auto-rotating-carousel'
Like this.

Reply:
ah, ok, good practice